static/app/components/group/ownedBy.tsx

```diff
@@ -68,25 +68,26 @@ function OwnedBy({group, project, organization}: OwnedByProps) {
           />
         </TitleWrapper>
 
-        <ActionsWrapper>
+        <ButtonBar>
```

Comment:
buttonbar is overkill imo

Reply:
isn't this exactly what this is for tho?

Reply:
button bar is for marking things active? Its got like 100 lines of styles compared to a flexbox with a gap

Reply:
What you mean marking as active? ButtonBar is just for a set of buttons that are next to each other no?